web/app/dashboard/dashboard.component.html

```diff
@@ -25,7 +25,7 @@
         <ng-container matColumnDef="latestBuild">
           <mat-header-cell *matHeaderCellDef>Latest Build</mat-header-cell>
           <!-- TODO: Pipe this into a relative time pipe -->
-          <mat-cell *matCellDef="let project">{{project.latestDate | date: 'fullDate'}}</mat-cell>
+          <mat-cell *matCellDef="let project">{{ (project.latestDate !== null) ? (project.latestDate | date: 'fullDate') : "n/a"}}</mat-cell>
```

Comment:
shouldn't need to specify null, we can just do `project.latestDate ?  (project.latestDate | date: 'fullDate') : "n/a"`

What do you think about an em dash `—` instead of n/a?